geniza/corpus/tests/test_corpus_templates.py

```diff
@@ -136,6 +136,16 @@ def test_editors(self, client, document, source, twoauthor_source):
         response = client.get(document.get_absolute_url())
         assertContains(response, "Editors")
 
+    def test_multiple_shelfmarks(self, client, document, join):
```

Comment:
please add comments or separate into two tests to clarify that one _should_ display the shelfmark as a field and one should not

Reply:
9f8c3ee ✅